pom.xml

```diff
@@ -84,40 +84,20 @@ THE SOFTWARE.
         <tag>${scmTag}</tag>
     </scm>
 
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-${jenkins.baseline}.x</artifactId>
-                <version>3893.v213a_42768d35</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
-        </dependencies>
-    </dependencyManagement>
-
-    <dependencies>
-        <dependency>
-            <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>timestamper</artifactId>
-            <scope>test</scope>
-        </dependency>
-    </dependencies>
```

Comment:
As `timestamper` is never used I removed it. The `bom` dependency becomes obsolete.

Reply:
I added the Timestamper plugin with 7137dbde28f301a994f6294f726419725577da67 to ease testing that the collapsing section panel is the last panel in the sidebar. So that was solely to ease development while I worked on the fix.